### DISCUSSION TITLE
Vercel and Cloudflare

### DISCUSSION THREAD

**Discussion by IVEN21** [vercel/next.js#67632] [Help]
### Summary

I host a website on Vercel, my audiences are global. Is it recommended to also use Cloudflare for additional services like security protection and performance enhancements?

**Comment by BorhanSaflo** [comment]
Vercel already uses its [Edge Network](https://vercel.com/docs/edge-network/overview), which functions both as a CDN and a globally distributed platform. This setup caches content, offering the same benefits you'd get from Cloudflare. Adding another CDN on top of the existing one might actually increase latency and response times.

Moreover, Vercel provides [DDoS Mitigation](https://vercel.com/docs/security/ddos-mitigation) to protect your site from such attacks. Considering these features, there's little advantage to using Cloudflare alongside Vercel.
